docling/models/picture_description_api_model.py

```diff
@@ -24,13 +24,15 @@ def get_options_type(cls) -> Type[PictureDescriptionBaseOptions]:
     def __init__(
         self,
         enabled: bool,
+        description_type: str,
```

Comment:
if we decide (still to be discussed) to go this way, we should use an Enum here, or directly the document type.